common/models/user.js

```diff
@@ -428,6 +428,117 @@ module.exports = function(User) {
   };
 
   /**
+   * Default verification options to use when calling User.verify()
+   * This is the full list of possible params, with example values
+   *
+   * ```js
+   * var options = {
+   *      type: 'email',
+   *      mailer: anEmailModel,
+   *      to: 'test@email.com',
+   *      from: 'noreply@email.com'
+   *      subject: 'verification email subject',
+   *      text: 'verification email text',
+   *      headers: {'Mime-Version': '1.0'},
+   *      template: 'path/to/template.ejs',
+   *      redirect: '/',
+   *      verifyHref: '',
+   *      host: 'localhost'
+   *      protocol: 'http'
+   *      port: 3000,
+   *      restApiRoot= '/api',
+   *      tokenGenerator: function (user, cb) { cb("random-token"); }
+   *    };
+   *```
+   *
+   * Usually you should only require to modify a subset of these params
+   * See `User.verify()` and `User.prototype.verify()` doc for params reference
+   */
+
+  User.verifyOptions = {
```

Comment:
thanks for noting that. i was not sure about using a model property
i will move it as getVerifyOptions()